mofacts/client/views/home/profile.js

```diff
@@ -89,22 +93,54 @@ Template.profile.helpers({
 // Template Events
 
 Template.profile.events({
-
+  //Enable TDF
+  'click .enableTdf': function(event, instance) {
```

Comment:
this doesnt do anything. `enableTdf` doesn't reference anything.

Reply:
> Files in the "new_templates" folder should be moved to more appropriate locations.

No. I need them there to keep track of what I've done.  That was purposeful. Changing the UI is a pretty lengthy process, so I need to keep track of them. This doesn't effect the build whatsoever.

> this doesn't do anything. `enableTdf` doesn't reference anything.

It wil l.This doesn't effect the build whatsoever.  The point of these incremental updates was to test them. There will be times in the near future where I write code before it's UI is implemented. 

> Searching for a TDF breaks here. `profile.js:133 Uncaught TypeError: tdf.tdfs.tutor.setspec.tags.toLowerCase is not a function`

Cannot reproduce.  

![image](https://user-images.githubusercontent.com/46696077/231882144-6db9418a-d0e5-4c07-91a6-9c0c173af651.png)

Reply:
Actually, that last one I can. I don't have access to most of the TDFs, so I can't test tdfs with tags that are arrays. Let me fix that one right quick